websites/management/commands/update_content_fields.py

```diff
@@ -0,0 +1,63 @@
+"""Updates multiple fields of content based on starter"""
+from django.core.exceptions import FieldDoesNotExist
+from django.core.management import BaseCommand, CommandError
+from django.db import transaction
+
+from websites.models import WebsiteContent
+
+
+class Command(BaseCommand):
+
+    help = __doc__
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "starter",
+            help="The WebsiteStarter slug to process",
+        )
+
+        parser.add_argument(
+            "-t",
+            "--type",
+            dest="type",
+            default="testimonials",
+            help="The type of content to process",
+        )
+
+        parser.add_argument(
+            "-k",
+            "--keys",
+            dest="keys",
+            nargs="+",
+            default=["type", "dirpath"],
+        )
+
+        parser.add_argument(
+            "-d",
+            "--data",
+            dest="data",
+            nargs="+",
+            default=["stories", "content/stories"],
+        )
```

Comment:
Suggestion, to make sure nothing is in the wrong order:  get rid of the `keys` argument, and expect the `data` argument to be key-value pairs like this:

```
update_content_fields ocw-www --type testimonials --data type=stories dirpath=content/stories
```

then `updated_data` would change to 
```python
updated_data = {x: y for x, y in [kv.split("=") for kv in data]}
```